src/Hint/Fixities.hs

```diff
@@ -0,0 +1,79 @@
+{-
+
+Raise a warning if you have redundant brackets in nested infix expressions.
+
+<TEST>
+yes = 1 + (2 * 3) -- @Ignore 1 + 2 * 3
+yes = (2 * 3) + 1 -- @Ignore 2 * 3 + 1
+no = (1 + 2) * 3
+no = 3 * (1 + 2)
+no = 1 + 2 * 3
+no = 2 * 3 + 1
+yes = (a >>= f) >>= g -- @Ignore a >>= f >>= g
+no = (a >>= \x -> b) >>= g
+</TEST>
+-}
+
+module Hint.Fixities(fixitiesHint) where
+
+import Hint.Type(DeclHint,Idea(..),rawIdea,Severity(..),toSS)
```

Comment:
GHC says the import of Severity here is unused. Other than that, looks good to go.

Reply:
Apparently it's also being imported from `Config.Type` and I'm not getting the warning because I'm on an older GHC. Done